common.sh

```diff
@@ -203,6 +203,7 @@ function run_test {
     $ADB shell "input keyevent HOME"
     sleep 5
     $ADB shell "am force-stop ${package_name}"
+    sleep 1
```

Comment:
Can we add a comment explaining why this line is necessary and what impact it has on startup numbers so it's not forgotten if this is refactored? :)

Reply:
Done!